spec/lang/syntax.md

```diff
@@ -326,6 +326,13 @@ pub enum Terminator {
     },
     /// Return from the current function.
     Return,
+    /// Assert the condition if false panic else go `next_block`.
+    Assert {
```

Comment:
This should not be a primitive of the language, since it can be encoded with if-then-else and the "panic" intrinsic.

Reply:
No minimize translates the program with some if-then-else blocks. It is a bit hacky.